.github/workflows/canary.yml

```diff
@@ -7,8 +7,9 @@ on:
     types: [checks_requested]
 
 env:
-  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_TOKEN: ${{ secrets.ARI_ADMIN_PAT_TOKEN }}
```

Comment:
The naming here seems kinda strange, is it intentional?

Reply:
Yes. It is a temp personal token just to have higher permission to write PR and CHANGELOG. Once infra team figure out the best way to manage these tokens, it shall get replaced